.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,22 @@
+---
+-name: 🐛 Report a bug
+-about: Report a reproducible or regression bug in React Native CLI.
+---
```

Comment:
can you please add `labels` as well? see: https://github.com/facebook/jest/blob/master/.github/ISSUE_TEMPLATE/bug.md